pkg/resource/azurerm/testdata/acc/azurerm_lb_rule/.driftignore

```diff
@@ -0,0 +1,3 @@
+*
+!azurerm_lb
```

Comment:
do we really need to list lb here ? Why ?

Reply:
No, it's not mandatory, I can delete it